src/function/gds/all_shortest_paths.cpp

```diff
@@ -457,27 +461,33 @@ class VarLenJoinsAlgorithm final : public RJAlgorithm {
 
 function_set VarLenJoinsFunction::getFunctionSet() {
     function_set result;
-    result.push_back(std::make_unique<GDSFunction>(name, std::make_unique<VarLenJoinsAlgorithm>()));
+    auto algo = std::make_unique<VarLenJoinsAlgorithm>();
```

Comment:
I was trying to overload GDS function, but then I realize it might be too early to do so. But anyway not inline it lays the foundation for overloading later on.